### PR TITLE
Update prereqs.md

### PR DESCRIPTION
**Pull request by lucasdigital**
This version is a little AWS-biased. We could expand to include some specifics for Azure/GCS details but as they have not been mandated I thought it best to keep the section as small as possible.

### CODE REVIEW

File: docs/prereqs.md [hunk 1]
```diff
@@ -8,6 +8,37 @@ Ready to start? Check the prerequisites and then follow these steps to get ready
 
 [Read the release notes](https://community.wandisco.com/s/article/Release-Notes-WANdisco-LiveData-Migrator-Release-Notes-1-11-1) to get the latest information about the current version of LiveData Migrator.
 
+## Recommended Technical Knowledge
+
+Before starting this installation, you should consider the following knowledge requirements:
```
Comment (shonaisbister-disco) [inline docs/prereqs.md:13]: I think you can remove this altogether (to keep consistent with the same section in the use case content): https://docs.wandisco.com/live-data-migrator/docs/uc-on-prem-hadoop-aws-s3-glue#recommended-technical-knowledge

File: docs/prereqs.md [hunk 2]
```diff
@@ -8,6 +8,37 @@ Ready to start? Check the prerequisites and then follow these steps to get ready
...
+### Networking
+
+* IP Address assignation
+* TCP/IP ports and Firewall setup or server certificates (for SSL)
+
+### Cloud Storage Technologies
+
+* Proficiency with intended target storage technologies, such as ADLS Gen 2, GCS, HDFS, and Amazon Simple Storage Service (S3).
+* For Amazon Web Services, this includes:
+  - Knowledge of AWS Marketplace, AWS EC2 Console, Amazon Simple Storage Service (S3), and AWS Command Line Tool.
```
Comment (shonaisbister-disco) [inline docs/prereqs.md:32]: Line 32: spell out EC2 in full for first use - 'Amazon Elastic Compute Cloud (Amazon EC2)

File: docs/prereqs.md [hunk 3]
```diff
@@ -8,6 +8,37 @@ Ready to start? Check the prerequisites and then follow these steps to get ready
...
+* Service configuration and management
+
+### Networking
+
+* IP Address assignation
+* TCP/IP ports and Firewall setup or server certificates (for SSL)
+
+### Cloud Storage Technologies
+
+* Proficiency with intended target storage technologies, such as ADLS Gen 2, GCS, HDFS, and Amazon Simple Storage Service (S3).
```
Comment (shonaisbister-disco) [inline docs/prereqs.md:30]: Line 30: update parentheses to (Amazon S3)

(I promise I'm not being a pedant about this - feel like a jerk but it's from the doc from AWS!)

**Review by shonaisbister-disco — CHANGES_REQUESTED** [review]
Looks great, just some minor branding updates.

File: docs/prereqs.md (hunk 1 above)
Comment (lucasdigital) [reply docs/prereqs.md:13]: Removed introductory text, to match with the use case.

File: docs/prereqs.md (hunk 2 above)
Comment (lucasdigital) [reply docs/prereqs.md:32]: Agreed. Done!

File: docs/prereqs.md (hunk 3 above)
Comment (lucasdigital) [reply docs/prereqs.md:30]: Agreed. This was discussed and clarified on Friday but as this was established content I wasn't critical enough when moving it over. Now expanded.

**Pull request merged**